user/environment-variables.md

```diff
@@ -175,11 +175,17 @@ The following default environment variables are available to all builds.
 Additionally, Travis CI sets environment variables you can use in your build, e.g.
 to tag the build, or to run post-build deployments.
 
-- `TRAVIS_ALLOW_FAILURE`: Set to `true` if the job is allowed to fail. `false` if not.
-- `TRAVIS_BRANCH`: For builds not triggered by a pull request this is the
-  name of the branch currently being built; whereas for builds triggered
-  by a pull request this is the name of the branch targeted by the pull
+- `TRAVIS_ALLOW_FAILURE`:
+  + set to `true` if the job is allowed to fail.
+  + set to `false` is not allowed to fail.
+- `TRAVIS_BRANCH`:
+  + for builds not triggered by a pull request this is the
+  name of the branch currently being built.
+  + for builds triggered by a pull request this is the name of the branch targeted by the pull
```

Comment:
Although the default branch in GitHub is normally `master` maybe we can make this a bit more generic:

>in many cases this will be your default branch. 

wdyt?